README.md

```diff
@@ -7,6 +7,15 @@ the API.
 
 ## API Endpoints
 
+### Authentication
+
+The API uses token based authentication. A sample request looks like this:
+
+    curl 'http://localhost:3000/api/measurements' -H "Authorization: Bearer 0123456789ABCDEF" -v
+
+All index and show resources are publicly available while writing to the API is permitted only
+consumers who provide a private api key.
```

Comment:
You need an API key for reading as well. We register it as public API key on the `ApiConsumer`.

Reply:
Ah, good point. Yes, this text is simply wrong (and I'd like to keep the API consumers that way for now).